bug: drizzle error when db:push with base project.

### Provide environment information

 System:
    OS: Linux 6.2 Ubuntu 22.04.3 LTS 22.04.3 LTS (Jammy Jellyfish)
    CPU: (4) x64 Intel(R) Core(TM) i7-5600U CPU @ 2.60GHz
    Memory: 9.28 GB / 15.49 GB
    Container: Yes
    Shell: 5.8.1 - /usr/bin/zsh
  Binaries:
    Node: 20.5.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 9.8.0 - /usr/local/bin/npm

### Describe the bug

When creating a new project ```npx create-turbo@latest -e https://github.com/t3-oss/create-t3-turbo```
then installing with pnpm i and doing ```cp .env.example``` to .env

When I run pnpm db:push

```
> create-t3-turbo@ db:push /home/anon/helloLast
> pnpm -F db push


> @acme/db@0.1.0 push /home/anon/helloLast/packages/db
> drizzle-kit push:mysql

drizzle-kit: v0.19.12
drizzle-orm: v0.28.2

No config path provided, using default 'drizzle.config.ts'
Reading config file '/home/anon/helloLast/packages/db/drizzle.config.ts'
Ignoring invalid configuration option passed to Connection: sslaccept. This is currently a warning, but in future versions of MySQL2, an error will be thrown if you pass an invalid configuration option to a Connection
/home/anon/helloLast/node_modules/drizzle-kit/index.cjs:35555
      const createConnectionErr = new Error();
                                  ^

Error: getaddrinfo EAI_AGAIN host
    at createConnection2 (/home/anon/helloLast/node_modules/drizzle-kit/index.cjs:35555:35)
    at connectToMySQL (/home/anon/helloLast/node_modules/drizzle-kit/index.cjs:37662:60)
    at Command.<anonymous> (/home/anon/helloLast/node_modules/drizzle-kit/index.cjs:53100:28) {
  code: 'EAI_AGAIN',
  errno: -3001,
  sqlState: undefined
}

Node.js v20.5.0
/home/anon/helloLast/packages/db:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @acme/db@0.1.0 push: `drizzle-kit push:mysql`
Exit status 1
 ELIFECYCLE  Command failed with exit code 1
```

### Link to reproduction

https://github.com/t3-oss/create-t3-turbo

### To reproduce

```
npx create-turbo@latest -e https://github.com/t3-oss/create-t3-turbo

# Install dependencies
pnpm i

# Configure environment variables
# There is an `.env.example` in the root directory you can use for reference
cp .env.example .env

# Push the Drizzle schema to the database
pnpm db:push

```

### Additional information

_No response_

--- a/.env.example
+++ b/.env.example
@@ -4,8 +4,11 @@
 # This file will be committed to version control, so make sure not to have any secrets in it.
 # If you are cloning this repo, create a copy of this file named `.env` and populate it with your secrets.
 
-# The database URL is used to connect to the database
-DATABASE_URL='mysql://user:password@host/db?sslaccept=strict'
+# The database URL is used to connect to your PlanetScale database.
+# 1st string works for connecting to PlanetScale via `pscale connect DATABASE_NAME`.
+# For production, you should use a connection string that uses SSL which looks like the 2nd string.
+DATABASE_URL='mysql://root:password@127.0.0.1:3306/personal'
+# DATABASE_URL=mysql://USERNAME:pscale_pw_PASSWORD@aws.connect.psdb.cloud/DATABASE_NAME?ssl={"rejectUnauthorized":true}
 
 # @see https://next-auth.js.org/configuration/options#nextauth_url
 NEXTAUTH_URL='http://localhost:3000'

--- a/.env.example
+++ b/.env.example
@@ -7,7 +7,7 @@
 # The database URL is used to connect to your PlanetScale database.
 # 1st string works for connecting to PlanetScale via `pscale connect DATABASE_NAME`.
 # For production, you should use a connection string that uses SSL which looks like the 2nd string.
-DATABASE_URL='mysql://root:password@127.0.0.1:3306/personal'
+DATABASE_URL='mysql://root:password@127.0.0.1:3306/DATABASE_NAME'
 # DATABASE_URL=mysql://USERNAME:pscale_pw_PASSWORD@aws.connect.psdb.cloud/DATABASE_NAME?ssl={"rejectUnauthorized":true}
 
 # @see https://next-auth.js.org/configuration/options#nextauth_url

